string command
---
string figure
string color
